tools/find_untranslated_strings.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from collections import namedtuple
+
+import re
+
+TransAndKey = namedtuple("TransAndKey", "translation, key")
+
+
+translation = re.compile(r'([a-z]{2}|zh-Han[st])\s*=\s*.*$', re.S | re.MULTILINE)
+
+ITUNES_LANGS = ['en', 'ru', 'ar', 'cs', 'da', 'nl', 'fi', 'fr', 'de', 'hu', 'id', 'it', 'ja', 'ko', 'nb', 'pl', 'pt', 'ro', 'sl', 'es', 'sv', 'th', 'tr', 'uk', 'vi', 'zh-Hans', 'zh-Hant']
```

Comment:
They are sorted in accordance with the order they should appear in the resulting autoformatted strings.txt file.